Mathlib/Algebra/Ring/Equiv.lean

```diff
@@ -899,6 +871,29 @@ end RingEquiv
 
 namespace RingEquiv
 
+section NonUnital
+
+variable [NonUnitalNonAssocSemiring R] [NonUnitalNonAssocSemiring S]
+
+/-- If a non-unital ring homomorphism has an inverse, it is a ring isomorphism. -/
+@[simps]
+def ofNonUnitalRingHom (hom : R →ₙ+* S) (inv : S →ₙ+* R)
```

Comment:
Can you confirm what this simps generates? I think `symm_ofNonUnitalRingHom` should be simp and if this generates a `ofNonUnitalRingHom_symm_apply`, that should not be simp.